content/en/docs/tasks/policy-enforcement/enabling-policy/index.md

```diff
@@ -1,10 +1,23 @@
 ---
-title: Enabling Policy Enforcement
+title: Enabling Policy Enforcement (Deprecated)
 description: This task shows you how to enable Istio policy enforcement.
 weight: 1
 keywords: [policies]
 ---
 
+{{< warning >}}
+The mixer policy is deprecated in Istio 1.5 and not recommended for production usage.
+
+* Rate limiting: Consider using Envoy native rate limiting instead of mixer rate limiting.
```

Comment:
Link to https://www.envoyproxy.io/docs/envoy/v1.13.0/intro/arch_overview/other_features/global_rate_limiting